pages/home/oracle-integrity-staking/reward-examples.mdx

```diff
@@ -28,6 +36,21 @@ $$
 \end{aligned}
 $$
 
+<Image
+  src={OIS_Rewards_Example_Light_1}
+  alt="Example 1"
+  sizes="100%"
+  placeholder="blur"
+  className="dark:hidden"
+/>
+<Image
+  src={OIS_Rewards_Example_Dark_1}
+  alt="Example 1"
+  sizes="100%"
+  placeholder="blur"
+  className="hidden dark:block"
+/>
```

Comment:
You can wrap up this pattern into a component to reduce the amount of duplication:

```tsx
import type { ComponentProps } from "react";
import Image from "next/image";
import clsx from "clsx";

type ImageProps = ComponentProps<typeof Image>;
type Props = Omit<ImageProps, "src"> & {
  darkSrc: ImageProps["src"];
  lightSrc: ImageProps["src"];
}

const DualModeImage = ({ darkSrc, lightSrc, className, ...props }: Props) => (
  <>
    <Image src={lightSrc} className={"dark:hidden", className} {...props} />
    <Image src={darkSrc} className={"hidden dark:block", className} {...props} />
  </>
);


...


<DualModeImage
  lightSrc={OIS_Rewards_Example_Light_1}
  darkSrc={OIS_Rewards_Example_Dark_1}
  sizes="100%"
  placeholder="blur"
/>
```